recipes/dask-sql/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dask-sql" %}
+{% set version = "0.1.0rc2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_sql-{{ version }}.tar.gz
+  sha256: 683e4c9759ef0a0d475aadc42d976125a04f9cc7d32065d11f48bd27282c46a1
+
+build:
+  noarch: python
```

Comment:
Is this really noarch python, even with the Java bits?

Reply:
Thanks for the review @scopatz! 
I think "noarch" should be fine - at least this is what I expect from Java.
The "python" part also is not 100% clear to me, so thanks for asking. I understood the conda-forge documentation that there are no C++ compiled parts - which is true - but I do not know if also the java "compilation" counts here.
As it is basically a performance optimization on the build side I am also fine with removing the "python" part. Can/should I replace it with "noarch generic" then or remove this part completely?

Reply:
Is the jar just a data-file part of the python package?

Should be fine to regard it as noarch

Reply:
The jar is created during the [python package installation](https://github.com/nils-braun/dask-sql/blob/main/setup.py#L38) via a [call](https://github.com/nils-braun/dask-sql/blob/main/setup.py#L9) to `mvn` and then specified in the `setup.py` as `package_data` [link](https://github.com/nils-braun/dask-sql/blob/main/setup.py#L62).
I honestly do not know if this is the best way to do it... 
But it is not already included in the repo, if this was your question.